whitesands/code/modules/mob/living/simple_animal/hostile/whitesands/survivors.dm

```diff
@@ -40,7 +40,7 @@
 		/obj/effect/spawner/lootdrop/whitesands/survivor
 	)
 	del_on_death = 1
-	faction = list("hermit")
+	faction = list("hermit", "wasteland") // VOID edit
```

Comment:
If you want to keep these for your planets you better modularize them now

Reply:
aren't survivors already modularized by putting them in the whitesands folder?

Reply:
depends on if you want to keep them in the rebase or not

Reply:
Actually, don't worry about this. I'll do it directly on the rebase pr